README.md

```diff
@@ -4,53 +4,57 @@
 
 
 
-##### What I do?
+#### What I do?
 
-I aim to give users a new way to interact with Google Maps through engaging text-based conversational interfaces. 
+I aim to give users a new way to interact with Google Maps through engaging text-based conversational interfaces.
 
-##### How old am I?
+#### How old am I?
 
 I'm only a baby bot right now, I need you to feed me with logic, data and inspiration.
 
-##### What is the motivation behind building me?
+#### What is the motivation behind building me?
 
-The primary motivation of the developers of MapBot is to provide a playground to tech enthusiasts, both beginners and advanced to try algorithms, approaches and ideas while contributing to a real-life project. 
+The primary motivation of the developers of MapBot is to provide a playground to tech enthusiasts, both beginners and advanced to try algorithms, approaches and ideas while contributing to a real-life project.
 
-##### What I aspire to be one day?
+#### What I aspire to be one day?
 
 - I want to help users in the most comprehensive way.
 - I want to give 'geeks' a platform to try out all things 'cool'.
 
 ------
 
-##### Are you here for GSSoC 2020?
+#### Are you here for GSSoC 2020?
 
 Check out all related information [here](GSSoC.md)
 
 ------
 
-##### What are some pre-requisites?
+#### What are some pre-requisites?
 
--  MySQL 
-  - Install the community version of mySQL from the [official mySQL documentation page](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/). 
+- MySQL
+  - Install the community version of mySQL from the [official mySQL documentation page](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/).
   - Create root user credentials during installation.
   - Verify the installation, running the command  `mysql -uroot -p -hlocalhost` should open the mySQL monitor. (Enter the root password when prompted)
 - StanfordCoreNLP
-  - StanfordCoreNLP has a dependency on Java 8. `java -version` should complete successfully with version 1.8 or higher. 
+  - StanfordCoreNLP has a dependency on Java 8. `java -version` should complete successfully with version 1.8 or higher.
   - Windows- Download as a .zip file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
   - Linux and MacOS- Follow the instructions to download the file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
+- Docker (if you want to set me up the Docker way)
+  - Take a look at [this](https://docs.docker.com/install/) for detailed installation instructions for Docker on Windows, Linux and Mac systems.
+  - You won't need to download MySQL locally to make Docker work, but it's recommended as a pre-requisite to be able to debug programs outside Docker.
+  - Verify the installations by `docker --version` and `docker-compose --version`
```

Comment:
Add this before the previous point, so the Docker related setups are together

Reply:
Shifted at the right place.